sql/core/src/test/scala/org/apache/spark/sql/DynamicPartitionPruningSuite.scala

```diff
@@ -259,8 +268,14 @@ abstract class DynamicPartitionPruningSuiteBase
           |SELECT f.date_id, f.store_id FROM fact_sk f
           |JOIN dim_store s ON f.store_id = s.store_id AND s.country = 'NL'
         """.stripMargin)
-
-      checkPartitionPruningPredicate(df, true, false)
+      if (adaptiveExecutionOn) {
+        // When enabling both AQE and DPP, PlanAdaptiveDynamicPruningFilters rule will insert
+        // the DPP filters only when the reused broadcast exchange exists. Otherwise it will
```

Comment:
It seems the reason is: DPP + AQE ignores the `DYNAMIC_PARTITION_PRUNING_REUSE_BROADCAST_ONLY`. I think it's fine, but we need to explain it correctly.

Reply:
or we can add the test tag `DisableAdaptiveExecution` to skip this test with AQE, as it doesn't make sense to AQE.

Reply:
Seems most of the test changes are caused by the same reason, and we can use `DisableAdaptiveExecution`.